modules/travel_pay/app/services/travel_pay/client.rb

```diff
@@ -91,7 +97,14 @@ def get_claims(veis_token, btsss_token)
       symbolized_body = response.body.deep_symbolize_keys
       parse_claim_date = ->(c) { Date.parse(c[:modifiedOn]) }
 
-      { data: symbolized_body[:data].sort_by(&parse_claim_date).reverse! }
+      sorted_claims = symbolized_body[:data].sort_by(&parse_claim_date).reverse
+
+      {
+        data: sorted_claims.map do |sc|
+          sc[:claimStatus] = sc[:claimStatus].underscore.titleize
```

Comment:
can we add this transformation to the tests? it's not clear what this does

Reply:
I added an expectation to the test. 

This converts the status in this way: `CamelCase` => `Camel_Case` => `Camel Case`.